### DISCUSSION TITLE
Oqtane Meetup in Southern California but everyone is welcome

### DISCUSSION THREAD

**Discussion by GregStevenson** [oqtane/oqtane.framework#3629] [General]
I've started an [Oqtane meetup](https://www.meetup.com/socal-oqtane-meetup/events/) and our next [event](https://www.meetup.com/socal-oqtane-meetup/events/298445347/) will be **Thursday January 18th from 7-9pm PST**. It will be a zoom session so you can join from anywhere if you wish. @markdav-is  will be the main presenter for the January meeting. 

I would be grateful if you could support the meetup by joining the group. I'm hoping it can serve as template for more meetups in other time zones in the future. I'll be sending out the zoom meeting details next week.

**Comment by GregStevenson** [comment]
I've updated the meetup description with the zoom meeting link, but I'm also including it here. Hope you can join us.

Here is the [zoom link](https://us06web.zoom.us/j/86489647713?pwd=urQLjpzteeq9hSZwNDzNb5vzhMacso.1) for the event. I'll start it about 30 minutes early and there is no waiting room, go everyone can network before we start the actual meeting at 7pm.

**Reply by thabaum** [reply]
@GregStevenson Thank you for putting this together for the community. That was a great presentation provided by @markdav-is and a lot of fun!  Look forward to the next meetup.

Cheers!

**Comment by thabaum** [comment]
Thanks @GregStevenson I will see if I can drop in for a few to say hi!

**Comment by GregStevenson** [comment]
This [link](https://drive.google.com/file/d/1wQLqp6AVrxNn_0FqGPnDayqcMHFdP6JW/view?usp=sharing) has the @markdav-is presentation showing how to create a theme in Oqtane and in this case, he is using MudBlazor/Material design. @sbwalker you can upload the video elsewhere if you wish. 

The full meetup recording can also be made available if desired, but, apart form Mark's talk,  it is mainly informal discussions about current Oqtane issues and other topics. 

Sorry for the delay, had some audio issues I needed to mitigate in order to get a decent render. I think my onboard sound is giving out

**Comment by avazon** [comment]
Thank you for the video link with markdav-is presentation, very helpful. I will try to create a Telerik version of this.

**Reply by thabaum** [reply]
That video is super informative for theme building, lots of great tips!
